scripts/init.sh

```diff
@@ -1,11 +1,14 @@
 #!/usr/bin/env bash
 
 set -e
+source $HOME/.cargo/env
 
 echo "*** Initializing WASM build environment"
 
+RUST_NIGHTLY_VERSION=$(cat rust-toolchain)
+
 if [ -z ${WASM_BUILD_TOOLCHAIN+x} ]; then
-	WASM_BUILD_TOOLCHAIN=nightly
+	WASM_BUILD_TOOLCHAIN=$RUST_NIGHTLY_VERSION
 fi
```

Comment:
So this part allows the user to manually specify an alternative nightly version, right?

Is that a feature we want to support? If so, this looks good. If not, we should remove the WASM_BUILD_TOOLCHAIN variable from here and any other scripts that use it.

Reply:
This is a conversation I had with @tgmichel and its part of the Substrate Template. If you have the variable defined in your environment it will use it.

Reply:
Yeah, I think leaving that option open does not hurt. lgtm!

Reply:
Dumb question, what is the `+x` about? (in the if condition)